bokehjs/gulp/tasks/test.coffee

```diff
@@ -4,6 +4,9 @@ mocha = require "gulp-mocha"
 paths = require "../paths"
 utils = require "../utils"
 
+global.atob = require "atob"
+global.btoa = require "btoa"
```

Comment:
@mattpap ``atob`` and ``btoa`` aren't available in node so I have to add them manually. Is this the proper way to do it?